build-system/global-configs/README.md

```diff
@@ -42,11 +42,41 @@ If `build-system/global-configs/custom-config.json` exists at build time, its pr
 
 The resulting config is
 
-```
+```json
 {
   "canary": 1,
   "chunked-amp": 1,
   "version-locking": 0,
   "cdnUrl": "https://example.com/amp"
 }
 ```
+
+# client-side-experiments-config.json
```

Comment:
Now that this file has detailed explanations for `custom-config.json` and `client-side-experiments-config.json`, the top `Flags` section looks a little weird on its own.

Can we change its title from `Flags` to something like `{prod|canary}-config.json`, and add a sentence explaining the purpose of the two files (before the _"Please be aware that canary-config.json..."_ line)?

Okay to do in a follow up if you prefer.

Reply:
Done, PTAL

Reply:
Perfect. Thank you!